consensus/src/consensus/mod.rs

```diff
@@ -679,6 +681,11 @@ impl ConsensusApi for Consensus {
         }
     }
 
+    fn get_sink(&self) -> Option<Hash> {
+        // TODO: unwrap on virtual state read when staging consensus is implemented
+        self.virtual_processor.virtual_stores.read().state.get().unwrap_option().map(|state| state.ghostdag_data.selected_parent)
+    }
+
     fn get_sink_timestamp(&self) -> Option<u64> {
         // TODO: unwrap on virtual state read when staging consensus is implemented
```

Comment:
I think we can just unwrap now. Right @michaelsutton ?

Reply:
yup